airflow/providers/cncf/kubernetes/pod_template_file_examples/git_sync_v4_template.yaml

```diff
@@ -0,0 +1,95 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# This file is used for documentation purposes. Example can be found at docs/executor/kubernetes.rst
+#
+# [START git_sync_template]
```

Comment:
I don't think this one works properly. This refrence already exist in 
https://github.com/apache/airflow/blob/e93460383f287f9b2af4b6bda3ea6ba17ba3c08b/airflow/providers/cncf/kubernetes/pod_template_file_examples/git_sync_template.yaml#L20

If v4 requires new example then we need a new refrence and a new entry in the docs:

https://github.com/apache/airflow/blob/e93460383f287f9b2af4b6bda3ea6ba17ba3c08b/docs/apache-airflow/core-concepts/executor/kubernetes.rst#L123-L128


It's odd that the doc build didn't fail on refrence from two distinct files

Reply:
thanks, fixed